WcaOnRails/config/locales/en.yml

```diff
@@ -699,7 +699,7 @@ en:
       organizations: "Regional Organizations"
       faq: "Frequently Asked Questions"
       teams: "Teams, Committees, and Councils"
-      tutorial: "Competition Tutorial"
+      tutorials: "Tutorials"
```

Comment:
Maybe just rename this to education?

Reply:
I've renamed that to "Education resources"